packages/bigquery/src/table.js

```diff
@@ -826,7 +826,7 @@ Table.prototype.getRows = function(options, callback) {
  * //-
  * var metadata = {
  *   encoding: 'ISO-8859-1',
- *   sourceFormat: 'NEWLINE_DELIMITED_JSON'
+ *   format: 'JSON'
```

Comment:
Oh, I see now. Thank you, this is indeed a bug! I will send a PR with a fix that will honor both `format` and `sourceFormat`.